src/upload-string-values.js

```diff
@@ -0,0 +1,71 @@
+// -*- mode: js; indent-tabs-mode: nil; js-basic-offset: 4 -*-
+//
+// This file is part of Thingpedia
+//
+// Copyright 2019 The Board of Trustees of the Leland Stanford Junior University
+//
+// Author: Silei Xu <silei@cs.stanford.edu>
+//
+// See LICENSE for details
+"use strict";
+
+const Tp = require('thingpedia');
+const fs = require('fs');
+const FormData = require('form-data');
+
+module.exports = {
+    initArgparse(subparsers) {
+        const parser = subparsers.addParser('upload-string-values', {
+            addHelp: true,
+            description: "Upload a string value dataset to Thingpedia."
+        });
+        parser.addArgument('--file', {
+            required: true,
+            help: "The .tsv file of the string values to upload."
+        });
```

Comment:
Usually, when there is only one input file, that file becomes that the default argument, and also we allow the user to specify multiple files that get concatenated, or specify `-` for standard input.